Description: Loosen pest version requirement
Author: Abhinaba Rakshit <abhinaba.rakshit@oss.qualcomm.com>
Origin: other, Debian packaging
Last-Update: 2026-04-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: mink-idl-compiler/idlc_ast/Cargo.toml
===================================================================
--- mink-idl-compiler.orig/idlc_ast/Cargo.toml
+++ mink-idl-compiler/idlc_ast/Cargo.toml
@@ -10,7 +10,7 @@ description = "AST structure for IDL; co
 
 [dependencies]
 idlc_errors = {path = "../idlc_errors" }
-pest = "2.7.9"
+pest = ">= 2.7.4, < 2.9.0"
 pest_derive = "2.7.9"
 thiserror = "1.0.50"
 
